web/controllers/donation_goal_controller.ex

```diff
@@ -2,12 +2,18 @@ defmodule CodeCorps.DonationGoalController do
   use CodeCorps.Web, :controller
   use JaResource
 
+  import CodeCorps.Helpers.Query, only: [id_filter: 2]
+
   alias CodeCorps.DonationGoal
 
   plug :load_and_authorize_changeset, model: DonationGoal, only: [:create]
   plug :load_and_authorize_resource, model: DonationGoal, only: [:update, :delete]
   plug JaResource
 
+  def filter(_conn, query, "id", id_list) do
```

Comment:
What are your thoughts on doing this in one line?

`def filter(_conn, query, "id", id_list), do: id_filter(query, id_list)`